### DISCUSSION TITLE
bodyAttrs usage

### DISCUSSION THREAD

**Discussion by mirocklez** [nuxt/nuxt#9822] [Questions (Nuxt 2)]
Hi,
I have a question on the proper usage of `bodyAttrs`.
Let's say I have some body class set in `nuxt.config.js`, then I have a body class set in the `default layout` and another body class set in within a page.
As a result in the rendered page I will have set only the body class set withing the page and the other two (set in `nuxt.config.js` & `default layout`) will be lost.
Is there a proper way to save the classes alredy set in the `nuxt.config.js` and the `default layout` and just to add a new class in the page that is not replacing the other classes?
You can see in the demo below:
 - if you delete`bodyAttrs` in `index.vue` page the body class set in the `default layout` will appear
 - if you delete `bodyAtts` in the `default layout`  the body class set in the `nuxt.config.js` will appear

Reproduction link:
https://codesandbox.io/s/nuxt-body-attrs-is1pl

**Comment by fitzage** [comment]
I know this is kind of old, so hopefully you figured it out already. Answering in part for anyone else who stumbles across this.

Putting the values in arrays solves the problem.

So instead of `class: "page-class"` and `class: "global-class"` you can use `class: ["page-class"]` and `class: ["global-class"]`.

I'm not familiar enough with codesandbox to make it work there, but it works in Nuxt v2.15.8 running on my machine.
